server/html/views/home.html

```diff
@@ -385,5 +191,22 @@ <h3 class="mb-5 text-lg font-normal text-gray-500 dark:text-gray-400 whitespace-
       deleteHostBtn.disabled = true;
     }
   }
+
+  function onCloseModal() {
+    window.location = "/"
+  }
+
+  const monitoring_table = document.getElementById('monitoring');
```

Comment:
Since you already have some changes here to do, please change variable name to camelCase